scaleapi/api.py

```diff
@@ -30,7 +30,7 @@ def __init__(
     ):
         if api_key == "" or api_key is None:
             raise ScaleException("Please provide a valid API Key.")
-
+        
```

Comment:
Those extra tabs/spaces are generally auto-removed when you have the `black` package in your local (or via pre-commit package). Do you use them locally?